activesupport/test/core_ext/duration_test.rb

```diff
@@ -21,7 +21,11 @@ class << k; undef_method :== end
   end
 
   def test_instance_of
-    assert 1.minute.instance_of?(Fixnum)
+    if 1.class == Integer
+      assert 1.minute.instance_of?(Integer)
+    else
+      assert 1.minute.instance_of?(Fixnum)
+    end
```

Comment:
I don't think a comment is necessary: we've used the same pattern elsewhere, the relationship it's stating is pretty self-descriptive (`1.minute` claims to have the same class as the original `1`), and if someone changes it they'll introduce either a warning or a test failure on at least one Ruby version.